bench/librawspeed/decompressors/DeflateDecompressorBenchmark.cpp

```diff
@@ -26,7 +26,7 @@
 #include "common/RawImage.h"                   // for RawImage, RawImageData
 #include "io/Buffer.h"                         // for Buffer, Buffer::size_...
 #include "io/ByteStream.h"                     // for ByteStream
-#include "io/Endianness.h"                     // for Endianness, Endiannes...
+#include "io/Endianness.h"                     // for Endianness, Endianness...
```

Comment:
FWIW, this was not so much a typo, but the symbols got elided to fit in 80 col limit

Reply:
Thanks. I changed it so that it wouldn't trigger a false positive